src/appengine/handlers/cron/backup.py

```diff
@@ -47,36 +47,31 @@ def get(self):
         local_config.PROJECT_PATH).get('backup.bucket')
     if not backup_bucket:
       logs.log('No backup bucket is set, skipping.')
-      return 'OK'
+      return
 
     kinds = [
         kind for kind in ndb.Model._kind_map  # pylint: disable=protected-access
         if (not kind.startswith('_') and kind not in EXCLUDED_MODELS)
     ]
 
     app_id = utils.get_application_id()
-    timestamp = datetime.datetime.utcnow().strftime('%Y-%m-%d-%H:%M:%S')
-    output_url_prefix = (
-        'gs://{backup_bucket}/datastore-backups/{timestamp}'.format(
-            backup_bucket=backup_bucket, timestamp=timestamp))
-    body = {
-        'output_url_prefix': output_url_prefix,
-        'entity_filter': {
-            'kinds': kinds
-        }
-    }
+    timestamps = [
+        datetime.datetime.utcnow().strftime('%Y-%m-%d-%H:%M:%S'),
+        'latest',
+    ]
+
+    for timestamp in timestamps:
+      output_url_prefix = (
+          'gs://{backup_bucket}/datastore-backups/{timestamp}'.format(
+              backup_bucket=backup_bucket, timestamp=timestamp))
+      body = {
+          'output_url_prefix': output_url_prefix,
+          'entity_filter': {
+              'kinds': kinds
+          }
+      }
 
-    try:
       request = _datastore_client().projects().export(
           projectId=app_id, body=body)
       response = request.execute()
-
-      message = 'Datastore export succeeded.'
-      status_code = 200
-      logs.log(message, response=response)
-    except googleapiclient.errors.HttpError as e:
-      message = 'Datastore export failed.'
-      status_code = e.resp.status
-      logs.log_error(message, error=str(e))
-
-    return (message, status_code, {'Content-Type': 'text/plain'})
+      logs.log('Datastore export succeeded.', response=response)
```

Comment:
I think you remove the error so that cron failure gets seen more prominently in error reporting right ?

Reply:
Yep, if we get an exception we'll just let it show in error reporting.